packages/editor/src/components/post-card-panel/style.scss

```diff
@@ -34,14 +34,6 @@
 	}
 }
 
-.editor-post-card-panel__icon.is-sync {
-	fill: var(--wp-block-synced-color);
-
-	& + .editor-post-card-panel__title {
-		color: var(--wp-block-synced-color);
-	}
-}
-
```

Comment:
Should we remove is-sync and is-global from the component. That would be a nice cleanup.

Reply:
Oh, yes.. I'll create a follow up.

Reply:
https://github.com/WordPress/gutenberg/pull/66472